src/features/views/components/ViewDataTable/columnTypes/LocalBoolColumnType.tsx

```diff
@@ -38,7 +40,7 @@ export default class LocalBoolColumnType implements IColumnType {
   }
 
   handleKeyDown(
-    model: ViewDataModel,
+    useViewGrid: UseViewGridReturn,
```

Comment:
This name is confusing to me. The `useViewGrid` name implies that this is a function, but it's not, it's the value returned by a function of that same name.

I would rename to `viewGrid`.